catalog/to-consul/resource.go

```diff
@@ -295,6 +298,9 @@ func (t *ServiceResource) shouldTrackEndpoints(key string) bool {
 		return false
 	}
 
+	if t.LoadBalancerEndpointsSync {
+		return svc.Spec.Type == apiv1.ServiceTypeNodePort || svc.Spec.Type == apiv1.ServiceTypeClusterIP || svc.Spec.Type == apiv1.ServiceTypeLoadBalancer
```

Comment:
I think these lines could be simplified to not repeat the same "or" condition twice (here and below in the last return statement). For example, you could remove this `if` and modify the last `return` statement as follows:
```
return svc.Spec.Type == apiv1.ServiceTypeNodePort ||
       svc.Spec.Type == apiv1.ServiceTypeClusterIP ||
       (t.LoadBalancerEndpointsSync && apiv1.ServiceTypeLoadBalancer)
```

Reply:
fixed via b30f9a21e80ed0a478e69ea84d60e1fc8287da92

Reply:
Looks good, thanks!